### ISSUE TITLE
Namespace clashes due to SSO

### ISSUE THREAD

**Issue by lukegb** [SpongePowered/Ore#60]
We need some way of handling namespace clashes with SSO. Imagine the following:

1. User *A* registers on forums
2. User *A* visits Ore and logs in. An `OreUser` is created.
3. User *A* created an Org *B*. An `Organisation` is created.
4. User *B* registers on forums
5. User *B* visits Ore.
6. User *B* attempts to log in.
7. :boom:

**Comment by JBYoshi** [comment]
Wait, so I can make an organization named `Zidane` and lock @Zidane out of Ore? (Insert evil laugh here...)

**Comment by lukegb** [comment]
No, because Zidane already exists on Ore. If he didn't already exist that'd be a problem.

**Comment by JBYoshi** [comment]
Aw...

Just kidding. I had no intentions of really doing it.

**Comment by lukegb** [comment]
:100:

**Comment by YourMCGeek** [comment]
Maybe you could check for the title of _Org B_ if it equals a name on the forums then create an error message?

**Comment by JBYoshi** [comment]
Then what happens if User B is created after Org B? Discourse wouldn't check for that.

**Comment by TheDoctorsLife** [comment]
Why not make Org's have random user names in the DB with the name User A Decided being displayed instead of the username?

**Comment by lukegb** [comment]
Because `ore.spongepowered.org/aerbjklasetroiljaegtroiajdfnvkljaenbvfklaerhjig/reponame` is not a useful URL

**Comment by TheDoctorsLife** [comment]
What about making Orgs thier own seperate thing? eg. `ore.spongepowered.org/org/orgname/reponame`
